src/bin/client-demo.rs

```diff
@@ -335,12 +338,30 @@ fn main() {
     // Compute/report stats
     let mut max_of_maxes = 0.0;
     let mut total_txs = 0;
-    for (max, txs) in maxes.read().unwrap().iter() {
+    let mut nodes_with_zero_tps = 0;
+    let mut total_maxes = 0.0;
+    for (max, txs, sock) in maxes.read().unwrap().iter() {
+        println!("Node:{}, Max TPS: {:.2}", *sock, *max);
+        if *max == 0.0 {
```

Comment:
Use `txs == 0` here? exact compare with float makes me cringe, although probably ok here.

Reply:
I'll change it to txs == 0